src/vcpkg-test/files.cpp

```diff
@@ -198,6 +201,7 @@ namespace
         fs.create_symlink(target_file, target_symlink, ec);
         if (ec)
         {
+            INFO(ec.message());
             REQUIRE(is_valid_symlink_failure(ec));
```

Comment:
Maybe that `INFO` should only be emitted when it's not a valid symlink failure? That may be spammy.

Reply:
That's how catch works -- the INFO is buffered and only emitted on failure.